src/screens/AccountBackup.js

```diff
@@ -55,16 +60,18 @@ class AccountBackupView extends React.PureComponent {
     }
   }
 
-  componentWillUnmount() {
+  async componentWillUnmount () {
     const { accounts } = this.props;
     const selected =
       accounts.getNew().address && accounts.getNew().address.length
         ? accounts.getNew()
         : accounts.getSelected();
-    accounts.lockAccount(selected);
+    await accounts.lockAccount(selected);
```

Comment:
`lockAccount` is not an `async` function. That means `await` has no sense here.

Reply:
Thanks, I'll remove that.